hcloud/instances.go

```diff
@@ -46,7 +46,7 @@ func newInstances(client *hcloud.Client, addressFamily addressFamily, networkID
 
 // lookupServer attempts to locate the corresponding hcloud.Server for a given v1.Node
 // It returns an error if the Node has an invalid provider ID or if API requests failed.
-// It can return a nil hcloud.Server if no
+// It can return a nil [*hcloud.Server] if neither the [ProviderID] nor the Name matches an existing server.
```

Comment:
What does the `[ProviderID]` syntax here do? Shouldn't it be something like `node.Spec.ProviderID`? And if that does some kind of cool symbol linking in the generated docs, do we also want that for `node.Spec.Name`?

Reply:
Thats... not doing anything, I tried to set it to `[v1.Node.Spec.ProviderID]` but that did not work, and then forgot to remove the braces. The linking unfortunately only works for top level symbols.